command/rootchain/params.go

```diff
@@ -0,0 +1,203 @@
+package rootchain
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	"github.com/0xPolygon/polygon-edge/rootchain"
+	"github.com/umbracle/ethgo/abi"
+)
+
+const (
+	dirFlag           = "dir"
+	rootchainAddrFlag = "rootchain-addr"
+	eventABIFlag      = "event-abi"
+	methodABIFlag     = "method-abi"
+	localAddrFlag     = "local-addr"
+	payloadTypeFlag   = "payload-type"
+)
+
+var (
+	params = &rootchainParams{}
+)
+
+const (
+	defaultRootchainName = "rootchain.json"
+)
+
+var (
+	errInvalidLocalAddr   = errors.New("invalid local SC address")
+	errInvalidPayloadType = errors.New("invalid payload type")
+	errInvalidEventABI    = errors.New("invalid event ABI")
+	errInvalidMethodABI   = errors.New("invalid method ABI")
+	errUnableToReadConfig = errors.New("unable to read rootchain config")
+)
+
+type rootchainParams struct {
+	configPath    string
+	rootchainAddr string
+	eventABI      string
+	methodABI     string
+	localAddr     string
+	payloadType   uint64
```

Comment:
Just to get a context, what is this for? If it's not required we should add in command description that default is 0 :)

Reply:
It's used for packing / unpacking specific event payloads. Right now we only have a single value - `ValidatorSetPayload`, that's by default 0 (that's why it's not showing up)